css/global.css

```diff
@@ -0,0 +1,433 @@
+/* 
+ * SITE
+ */
+
+:root {
+  /* color variables */
+  /* Some experiments, can delete if we are sure we don't need them */
+  /* #3dac78, #74c69d, #95d5b2, #b7e4c7, #d8f3dc */
+  /* #5603AD, #8367C7, #7EDFA0, #B3E9C7, #C2F8CB */
+  /* #344E41, #3A5A40, #588157, #5A9F68, #BBD58E */
+  /* #06a778, #c6e4b5, #eaf4cb, #f1ea9b */
+  /* #3ab284, #23c686, #c5d893 */
+  
+  --color-primary: #5a9f68;
+  --color-secondary: #1d3853;
+  --color-tertiary: #bbd581;
+  --color-dark: #344E41;
+  
+  --color-white: #f7f9f0;
+  --color-grey: #7a9295;
+  --color-black: #010101;
+  
+  /* header specs */
+  --header-height: 6rem;
+  --logo-size: 4rem;
+  --logo-font: 2.5rem;
+
+  /* spacing variables */
+  --margin-wide: 1.5rem;
+  --margin-mid: 1rem;
+  --margin-narrow: 0.5rem;
+
+  --padding-wide: 1.5rem;
+  --padding-mid: 1rem;
+  --padding-narrow: 0.5rem;
+
+  /* type variables */
+  --type-scale: 1.25;
+  --line-height: 1.5;
+  --font-size: 1rem;
```

Comment:
do you think we need to define a custom default `font-size` or are we happy with just the browser default 16px?

Reply:
I'm not sure, but I created a variable either way in case we want to change things later it would be much faster to do so.